types.d.ts

```diff
@@ -1,134 +1,132 @@
 import dgram = require("dgram");
 
-declare module "hot-shots" {
-  export type Tags = { [key: string]: string } | string[];
-  export interface ClientOptions {
-    bufferFlushInterval?: number;
-    bufferHolder?: { buffer: string };
-    cacheDns?: boolean;
-    errorHandler?: (err: Error) => void;
-    globalTags?: Tags;
-    globalize?: boolean;
-    host?: string;
-    isChild?: boolean;
-    maxBufferSize?: number;
-    mock?: boolean;
-    port?: number;
-    prefix?: string;
-    sampleRate?: number;
-    socket?: dgram.Socket;
-    suffix?: string;
-    telegraf?: boolean;
-    useDefaultRoute?: boolean;
-  }
-
-  export interface ChildClientOptions {
-    globalTags?: Tags;
-    prefix?: string;
-    suffix?: string;
-  }
-
-  export interface CheckOptions {
-    date_happened?: Date;
-    hostname?: string;
-    message?: string;
-  }
-
-  export interface DatadogChecks {
-    OK: 0;
-    WARNING: 1;
-    CRITICAL: 2;
-    UNKNOWN: 3;
-  }
-
-  type unionFromInterfaceValues4<
-    T,
-    K1 extends keyof T,
-    K2 extends keyof T,
-    K3 extends keyof T,
-    K4 extends keyof T,
+export type Tags = { [key: string]: string } | string[];
```

Comment:
Removing `declare module` _seems_ harmless as the main reason typescript suggest using it is to describe multiple modules in a single `.d.ts` file.